packages/utils/src/utils/api/index.spec.ts

```diff
@@ -0,0 +1,168 @@
+import fetchMock from 'fetch-mock';
+import * as kununuSessionCallback from '@kununu/kununu-utils/dist/kununu-helpers/kununuSession';
+import {logger} from '@kununu/kununu-utils/dist/kununu-logger';
+
+import fetchApiDomain, {getBFFURL} from '../fetchApiDomain';
+
+import fetchApi from '.';
+
+jest.mock('../isClientRender', () => () => true);
+
+jest.mock('next/config', () => () => ({
+  publicRuntimeConfig: {
+    server: {name: 'app-profiles'},
+  },
+}));
+
+jest.mock('@kununu/kununu-utils/dist/kununu-logger', () => ({
+  logger: {
+    error: jest.fn(),
+    info: jest.fn(),
+  },
+}));
+
+jest.spyOn(kununuSessionCallback, 'default').mockImplementation(() => ({
+  default: jest.fn(),
+}));
+
+describe('Utils', () => {
+  describe('api', () => {
+    beforeEach(() => {
+      process.env.BFF_URL = 'http://someurl.com';
+    });
+
+    afterAll(() => {
+      delete process.env.BFF_URL;
+    });
```

Comment:
I think it will work without this, since we have this variable mocked in a global mock